src/graphql/mongoose_graphql_account_helper.erl

```diff
@@ -0,0 +1,26 @@
+-module(mongoose_graphql_account_helper).
+
+-export([format_result/2, make_error/2, make_error/3]).
+
+-include("mongoose_graphql_types.hrl").
+
+-spec format_result(InResult, Context) -> OutResult when
+      InResult :: {atom(), string() | binary() | integer()},
+      Context :: map(),
+      OutResult :: {ok, binary() | integer()} | {error, resolver_error()}.
+format_result(Result, Context) ->
+    case Result of
+        {ok, Msg} when is_list(Msg) -> {ok, iolist_to_binary(Msg)};
+        {ok, Msg} -> {ok, Msg};
```

Comment:
Not a must, but I think it would look cleaner if in these functions we take advantage of the fact that `iolist_to_binary` works for binaries as well:

```suggestion
        {ok, Msg} -> {ok, iolist_to_binary(Msg)};
```

Reply:
I changed it a little to emphasize that value can also be an integer